health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/appConfigHelpers.js

```
@@ -143,7 +143,7 @@ const addValidationArrayToConfig = (field, fieldTypeMasterData = []) => {
 
 export const restructure = (data1, fieldTypeMasterData = [], parent) => {
   return[
-    ...data1.sort((a, b) => a.order - b.order)
+    ...data1?.sort((a, b) => a.order - b.order)
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)